### DISCUSSION TITLE
GitHub version of Buzz crashes on MacOS

### DISCUSSION THREAD

**Discussion by celik97** [chidiwilliams/buzz#635] [Q&A]
Greetings,
I downloaded Buzz with homebrew on my Macbook Air M2. The app opens without a problem. When I put the file I want to transcribe to it, it downloads the model and then crashes. I tried it on my friend's Macbook Pro M2 Pro, thinking my computer may not be powerful enough. But the same happened there also. 

Is this a problem on my part or is something wrong with Buzz?

This the the first part of the crash report:

-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Buzz [25856]
Path:                  /Volumes/*/Buzz.app/Contents/MacOS/Buzz
Identifier:            com.chidiwilliams.buzz
Version:               0.8.4 (???)
Code Type:             X86-64 (Translated)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-11-22 13:34:54.7095 +0300
OS Version:            macOS 14.1.1 (23B81)
Report Version:        12
Anonymous UUID:        8BF780FB-16E7-1E27-9A62-5C4CDA2BFC75

Sleep/Wake UUID:       C83B881A-4682-4F88-B6F7-BEBE3C5D8263

Time Awake Since Boot: 190000 seconds
Time Since Wake:       99294 seconds

System Integrity Protection: enabled

Notes:
PC register does not match crashing frame (0x0 vs 0x142DB1E26)

Crashed Thread:        11  QThread

Exception Type:        EXC_BAD_INSTRUCTION (SIGILL)
Exception Codes:       0x0000000000000001, 0x0000000000000000

Termination Reason:    Namespace SIGNAL, Code 4 Illegal instruction: 4
Terminating Process:   exc handler [25856]

Error Formulating Crash Report:
PC register does not match crashing frame (0x0 vs 0x142DB1E26)

**Comment by jordiprenafeta** [comment]
I have the opposite problem: in my Macbook Pro M2 Pro Max the Appstore versions works faster but crashes every time I try to delete a transcription, and the Github versions works slower but with no chashes. Hope they fix all crashes soon!
